src/components/dropdown/dropdown.js

```diff
@@ -179,7 +202,7 @@ class Dropdown extends mixin(createComponent, initComponentBySearch, trackBlur)
    * Closes the dropdown menu if this component loses focus.
    */
   handleBlur() {
-    this.element.classList.remove('bx--dropdown--open');
+    this._toggle({ type: 'blur' });
```

Comment:
Kind of nit-picky, but `handleBlur()` takes actual event in the first arg, and I’d rather use it for the sake of consistency. The event has `focusin` or `focus` type.